lib/utils/routes.js

```diff
@@ -44,6 +45,10 @@ function routes(server) {
     res.end('</body></html>');
 
     function writeDirectory(baseUrl, basePath) {
+      if (isWebpack5 && baseUrl === 'auto') {
+        baseUrl = '';
+      }
```

Comment:
I'm not sure how to handle the `auto` public path. Is there a way to retrieve it?

Reply:
No need to check on webpack@5, `auto` means no `publicPath` (same as empty string or `/` if you from web page)

Reply:
@evilebottnawi So this (emptying the `baseUrl`) would work, right? Would it be impossible for webpack@4 to pass `auto` as the public path?

Reply:
> So this (emptying the baseUrl) would work, right?

Yes

> Would it be impossible for webpack@4 to have auto as the public path?

In theory yes, but I want to do as limitation for v4 and webpack@4 and write about it in changelog

Reply:
Removed webpack@5 check.